apps/reconstruct/reconstruct.cpp

```diff
@@ -445,6 +451,13 @@ int main(int argc, const char* argv[]) {
       skip = (*vec)[fp_i].value();
     }
   }
+
+  // Check skip_glass_roof_attribute: set only if skip is still false
+  if (auto vec = attributes.get_if<bool>(skip_glass_roof_attribute_name)) {
+    if ((*vec)[fp_i].has_value() && skip == false) {
```

Comment:
Here we take into account the 'skip_glass_roof_attribute_name' value ONLY if skip is false. If skip is already true because of the top10nl classification (or other reason) we don't want it to go back to false because of the glass-roof-detection method